docs/guide/conditional-validation.md

```diff
@@ -37,10 +37,10 @@ $rules = [
 $result = (new Validator())->validate($data, $rules);
 ```
 
-Note that this setting must be set for every rule that needs to be skipped on error.
+Note that this setting must be set for each rule that needs to be skipped on error.
```

Comment:
`every` is used intentionally to avoid confusion with `Each` rule.

Reply:
> Every is similar to each in that both words are used to reference groups of items. While each can be used for groups of two or more, every refers to groups of at least three items. Additionally, every refers to the collective whole, not just the individuals.

https://www.grammarly.com/blog/each-and-every/